Worktez/worktez
# fix(Milestone):Fixed Milestone Add Task

### Functionality:
Fixed Milestone Component

### Solution:
- Fixed Milestone Add Task
- Fixed Task Data refresh on Create new Task

### Risk level:

### How to test:
Try Adding tasks to milestone, check whether it is getting added.
Create a new task and check whether it is visible in task tracker
